backend/service/auditsink/slack/slack_helper.go

```diff
@@ -0,0 +1,28 @@
+package slack
+
+import (
+	auditv1 "github.com/lyft/clutch/backend/api/audit/v1"
+	configv1 "github.com/lyft/clutch/backend/api/config/service/auditsink/slack/v1"
+	"github.com/lyft/clutch/backend/middleware"
+)
+
+// GetSlackOverrideText returns the custom slack message from the slack override config if the FullMethod matches
+// the serive and method in the audit event. Otherwise ok is false and an empty string is returned.
+func GetSlackOverrideText(override *configv1.Override, event *auditv1.RequestEvent) (string, bool) {
```

Comment:
let's hang this method off of the sink and store the overrides on the sink's struct.

Reply:
added Overrides to the sink's `svc` struct

RE "hang this method off of the sink" - can you clarify a bit more? Jst want to make sure I understand. For example, for the e2e flow, I was planning on calling the func in https://github.com/lyft/clutch/blob/main/backend/service/auditsink/slack/slack.go#L64 like so:
```
// Check if custom slack message provided in the config
	customMessage, ok := GetOverrideMessage(s.overrides, event)
	if ok {
		// call the slack transformer
	}
```

Reply:
oh, perhaps you meant changing it to a receiver function:
`func (s *svc) GetOverrideMessage (event *auditv1.RequestEvent)`, though then I wouldn't be able to reuse it in the internal implementation.

But I don't mind writing another func for the Lyft use-case.

Reply:
what i would do is write a helper func that generates an lookup table (LUT) from the []CustomMessage, e.g. `func(overrides []*CustomMessage) map[string]CustomMessage`. call that during init and store it on the service struct. then the lookup is trivial (and efficient) in anyone who generates an LUT from overrides using the helper.

Reply:
one more thought, if you want to be able to call GetOverride(service, method string) rather than combining the service and method string from event you can have the helper generate a OverrideLookup struct that keeps the map internally and exposes that func, then store the OverrideLookup func on the service struct

Reply:
thank you for the feedback! went with the 2nd approach and added new unit tests